test/e2e/load_test.go

```diff
@@ -161,4 +161,21 @@ var _ = Describe("Podman load", func() {
 		inspect.WaitWithDefaultTimeout()
 		Expect(result.ExitCode()).To(Equal(0))
 	})
+
+	It("podman load localhost repo", func() {
+		outfile := filepath.Join(podmanTest.TempDir, "load_test.tar.gz")
+		// this file was prepared with no repo tag
+		fetch := podmanTest.SystemExec("wget", []string{"-O", outfile, "https://github.com/projectatomic/libpod/files/1944502/load_test.tar.gz"})
+		fetch.WaitWithDefaultTimeout()
+		Expect(fetch.ExitCode()).To(Equal(0))
+
+		load := podmanTest.Podman([]string{"load", "-i", outfile})
+		load.WaitWithDefaultTimeout()
+		Expect(load.ExitCode()).To(Equal(0))
+
+		result := podmanTest.Podman([]string{"images", "-f", "label", "load:test"})
+		result.WaitWithDefaultTimeout()
+		Expect(result.LineInOutputContains("docker")).To(Not(BeTrue()))
+		Expect(result.LineInOutputContains("localhost")).To(BeTrue())
+	})
```

Comment:
Add a test of 
podman create --name test -ti fedora sleep 1
podman export -o test.tar test
podman rm test
podman load test.tar
podman images -f label load:test

Reply:
I am getting 
```
error pulling "dir:test.tar": unable to pull dir:test.tar: unable to find image in the registries defined in "/etc/containers/registries.conf"
```
for both this branch and master. Does export miss something that load needs? export/import works and save/load works (in that they don't fail) but not export/load

Reply:
that may expose a different issue ... this issue was caused by save.  maybe a different PR would be appropriate should the sequence @rhatdan suggests not work.  just my two cents

Reply:
@baude @rhatdan doing that sequence in docker also does not work, but the error is different. Should I still figure out why this sequence doesn't work?

Reply:
There's a mix of `export` (container ops) and `load` (image ops). I think the `load` above should be `import`.

Reply:
I always get confused on the combo. Should be podman save, I believe.

podman save 
podman load

podman extract
podman import

Reply:
So save isn't ready for this change to load, as load doesn't work when I save an image, yet it works when I use @edsantiago 's tar. I need to poke around more in save to figure out why it isn't saving with the correct format for load with localhost prepended. I think it has something to do with PushImage

Reply:
Well we need these two commands work together.
We also want 

podman save 
docker load

docker save 
podman load